### PR TITLE
Replace Element references with the actual Tchap information

### PR DESCRIPTION
**Pull request by yostyle**
#1

### CODE REVIEW

File: AUTHORS.md [hunk 1]
```diff
@@ -1,45 +1,23 @@
-A full developer contributors list can be found [here](https://github.com/vector-im/element-android/graphs/contributors).
+A full developer contributors list can be found [here](https://github.com/dinsic-pim/tchap-android-v2/graphs/contributors).
```
Comment (giomfo) [inline AUTHORS.md:1]: Don't you think better to link to the element-android authors: "https://github.com/vector-im/element-android/blob/master/AUTHORS.md"?

File: AUTHORS.md [hunk 2]
```diff
@@ -1,45 +1,23 @@
...
+# Tchap team:
 
 Even if we try to be able to work on all the functionalities, we have more knowledge about what we have developed ourselves.
 
-## [Benoit](https://github.com/bmarty): Android team leader
+## [Giomfo](https://github.com/giomfo): Product manager, Android team leader
 
-[@benoit.marty:matrix.org](https://matrix.to/#/@benoit.marty:matrix.org)
-- Android team leader and project leader, Android developer, GitHub community manager.
+[@Giom.marty:matrix.org](https://matrix.to/#/@Giom:matrix.org)
```
Comment (giomfo) [inline AUTHORS.md:9]:
```suggestion
[@Giom:matrix.org](https://matrix.to/#/@Giom:matrix.org)
```

File: TCHAP_CHANGES.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+Changes in Tchap 2.0.0 (2021-xx-xx)
+===================================================
+
+First release!
+
+
+=======================================================
++        TEMPLATE WHEN PREPARING A NEW RELEASE        +
+=======================================================
```
Comment (giomfo) [inline TCHAP_CHANGES.md:9]: Please move the template at the top, if you agree

Comment (yostyle) [reply TCHAP_CHANGES.md:9]: What do you mean by at the top ?

File: AUTHORS.md (hunk 1 above)
Comment (yostyle) [reply AUTHORS.md:1]: Why not. What's happen if we have other contributors on Tchap v2 ?

Comment (giomfo) [reply AUTHORS.md:1]: we should put both:
A full element-android developer contributors list can be found [here](https://github.com/vector-im/element-android/graphs/contributors).
A full tchap-android developer contributors list can be found [here](https://github.com/dinsic-pim/tchap-android-v2/graphs/contributors).

File: TCHAP_CHANGES.md (hunk 3 above)
Comment (giomfo) [reply TCHAP_CHANGES.md:9]: Ignore this remark, I misunderstood the meaning of this template

**Review by giomfo — APPROVED** [review]
Approved. I let you adjust the AUTHORS.md as you prefer

File: AUTHORS.md (hunk 1 above)
Comment (yostyle) [reply AUTHORS.md:1]: tchap-android developer contributors list includeselement-android developer contributors list

Comment (giomfo) [reply AUTHORS.md:1]: ok - ignore my remark ;)

**Pull request merged**